package-lock.json

```diff
@@ -21675,7 +21675,7 @@
     },
     "ignore-walk": {
       "version": "3.0.1",
-      "resolved": "",
+      "resolved": false,
```

Comment:
Not a big deal, but generally speaking please avoid pushing changes to the lock file that aren't necessary.